app/schemas/models/campaign.schema.js

```diff
@@ -122,7 +122,19 @@ _.extend(CampaignSchema.properties, {
       }
 
     }
-  }
+  },
+  scenarios: {
+    type: 'array',
+    title: 'AI Scenarios',
+    items: {
+      type: 'object',
+      // key is the scenario id property
+      properties: {
+        scenario: c.objectId({ links: [{ rel: 'extra', href: '/db/ai_scenario/{($)}' }] }),
```

Comment:
To make it cleaner/more understandable, can we rename `scenario` it to `id` or `scenarioId`?
Also where are the modules defined? Modules won't have names, just index numbers?